package.json

```diff
@@ -34,6 +34,7 @@
     "@polymer/iron-input": "^3.0.1",
     "@polymer/iron-label": "^3.0.1",
     "@polymer/iron-media-query": "^3.0.1",
+    "@polymer/iron-overlay-behavior": "^3.0.0-pre.27",
```

Comment:
3.0.2 is the latest version https://www.npmjs.com/package/@polymer/iron-overlay-behavior

Reply:
Yeah well let me see 1 Travis build passing and I'll change this to later versions and see if it works. So far this hasn't happened. Yarn.lock has this pre.27 version and when I do yarn add it adds the new version but doesn't remove the old one...